requirements/tests

```diff
@@ -9,7 +9,7 @@ pytest-asyncio
 httpx; python_version >= '3.6'
 uvicorn; python_version >= '3.6'
 aiofiles; python_version >= '3.6'
-daphne; python_version >= '3.6'
+daphne; python_version >= '3.6' and sys_platform != 'win32'
```

Comment:
We should also add `python_version >= '3.6'` to the `cbor` requirement below, IIRC it is only tested in the WebSocket context.